### PR TITLE
add algo builder to development list

### PR DESCRIPTION
**Pull request by vuvth**
# Description

- Rename #Development-JavaScript to #Development-JavaScript/TypeScript 
- Add algo-builder and algo-builder-templetes.
# Checklist:

_Please make sure you reviewed the checklist and comply with each requirement:_

-   [ ] My code follows the [contribution guidelines](../CONTRIBUTING.md) of this project
-   [ ] This pull request has a title in the format `Add Name of Resource`:
    -   [ ] Add `algo-builder`
    -   [ ] Add `algo-builder-templetes`

Please review my PR. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,10 +113,12 @@ Algorand is an open-source, proof of stake blockchain and smart contract computi
 - [tinyman-py-sdk](https://github.com/tinymanorg/tinyman-py-sdk) - Tinyman Python SDK.
 - [algofi-py-sdk](https://github.com/Algofiorg/algofi-py-sdk) - Algofi Python SDK.
 
-### JavaScript
+### JavaScript/TypeScript
```
Comment (aorumbayev) [inline README.md:116]:
```suggestion
### JavaScript & TypeScript
```

File: README.md [hunk 2]
```diff
@@ -113,10 +113,12 @@ Algorand is an open-source, proof of stake blockchain and smart contract computi
...
 
 - [js-algorand-sdk](https://github.com/algorand/js-algorand-sdk) - The Algorand JavaScript SDK & Examples.
 - [algofi-js-sdk](https://github.com/Algofiorg/algofi-js-sdk) - Algofi JavaScript SDK.
+- [Algo Builder](https://github.com/scale-it/algo-builder) - Framework to automate development of Algorand Assets and Smart Contracts.
```
Comment (aorumbayev) [inline README.md:120]:
```suggestion
- [algo-builder](https://github.com/scale-it/algo-builder) - Framework to automate development of Algorand Assets and Smart Contracts.
```

File: README.md [hunk 3]
```diff
@@ -113,10 +113,12 @@ Algorand is an open-source, proof of stake blockchain and smart contract computi
...
+- [Algo Builder Templates](https://github.com/scale-it/algo-builder-templates) - Dapps templates for Algo Builder
```
Comment (aorumbayev) [inline README.md:121]:
```suggestion
- [algo-builder-templates](https://github.com/scale-it/algo-builder-templates) - Dapps templates for Algo Builder.
```

**Review by aorumbayev — CHANGES_REQUESTED** [review]
Please review/apply suggested changes and once CI pipeline passes we can merge the pr. Thanks for contributing

**Comment by vuvth** [comment]
> Please review/apply suggested changes and once CI pipeline passes we can merge the pr. Thanks for contributing

Thank for your review! I applied the suggestions.

**Comment by aorumbayev** [comment]
> > Please review/apply suggested changes and once CI pipeline passes we can merge the pr. Thanks for contributing
> 
> Thank for your review! I applied the suggestions.

You are welcome man and thanks for awesome contributions! <3

**Pull request merged**